cypress/e2e/base.spec.ts

```diff
@@ -10,4 +10,23 @@ describe('Open browser and check if login button exists.', () => {
   });
 });
 
+describe('Visit landing page and verify users exists', () => {
+  beforeEach(() => {
+    cy.intercept('/api/users', { fixture: 'users.json' }).as('getUsers');
```

Comment:
I'm not confident about my attempt to intercept `/api/users` with data that doesn't exactly match what is being sent. The reason I kept it simplified is I expect what is sent to the client to change a fair bit of times throughout the process, and I was cautious about creating a response much more than what was necessary for this specific issue.